src/lib/components/NavBar.svelte

```diff
@@ -88,9 +89,7 @@
 
 
 <nav id="navbar" use:melt={$root}>
-  {#if !vendored}
-    <NavBanner/>
-  {/if}
+  <NavBanner {vendored}/>
```

Comment:
Do you still need this change? It seems like it would "just work" with the updated code.

Reply:
Yeah, I don't really care about the existence of those 30px, so I would say get rid of it entirely. What I figured the problem was, was the gap underneath the NavBar, which I think is fixed now irrespective of which way the NavBanner ultimately works in vendored mode.

I would say drop it entirely though if its simple.